From 7932bf28f912d1b9c347eafe50ffc1a8fd22161f Mon Sep 17 00:00:00 2001
From: Vinay Kumar Pabboju <vinayx.kumar.pabboju@intel.com>
Date: Mon, 22 Aug 2022 05:40:46 +0000
Subject: [PATCH] OOB Bios config not POR for ctl hence removing it

Excluding biosconfig commands functionality as it is not POR for ctl.

Tested:
Confirmed that for GetPayload command which is unsupported command
the BMC returns response C1h.

Signed-off-by: Vinay Kumar Pabboju <vinayx.kumar.pabboju@intel.com>
Upstream-Status: Pending
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ee58258..e78f9e8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,7 +104,7 @@ add_custom_command(OUTPUT include/ipmi-whitelist.hpp
                   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
 add_library (zinteloemcmds
-             SHARED src/oemcommands.cpp src/sensorcommands.cpp src/biosconfigcommands.cpp
+             SHARED src/oemcommands.cpp src/sensorcommands.cpp
              src/storagecommands.cpp src/multinodecommands.cpp
              src/firmware-update.cpp src/appcommands.cpp src/smbioshandler.cpp
              src/smbiosmdrv2handler.cpp src/manufacturingcommands.cpp
-- 
2.17.1

